ui/client/containers/Auth/StrategyInitializer.tsx

```diff
@@ -84,7 +84,7 @@ export function StrategyInitializer(props: PropsWithChildren<Props>): JSX.Elemen
           },
           buttonLabel: t("auth.StrategyInitializer.errors.504.buttonLabel", "Go to authentication page"),
           description: t(
-            "auth.StrategyInitializer.errors.504.description",
+            "auth.StrategyInitializer.errors.1024.description",
```

Comment:
Why did you change it?

Reply:
`ACCESS_TOKEN_CODE` is `1024` not `504` which is `HTTP_TIMEOUT_CODE`
there was two translation keys with different default value. this mistake appeared when changed extract method.